Rubberduck.RegexAssistant/Quantifier.cs

```diff
@@ -0,0 +1,90 @@
+﻿
+using System;
+using System.Text.RegularExpressions;
+
+namespace Rubberduck.RegexAssistant
+{
+    public class Quantifier
+    {
+        public static readonly Regex QuantifierMatch = new Regex(@"(?<!\\)[\?\*\+]|\{(\d+)(,\d*)?\}");
+        private static readonly Regex _expressionPattern = new Regex(@"^\{(?<min>\d+)(?<max>,\d*)?\}$");
```

Comment:
This is slightly different than the Atoms. Atoms are implemented vastly superior, because they allow reanchoring the pattern we have. Adapt this